core/src/main/java/hudson/model/View.java

```diff
@@ -172,8 +171,6 @@
      */
     protected boolean filterQueue;
     
-    protected transient List<Action> transientActions;
```

Comment:
NB: this is a backwards incompatible API change (however a search of public sources in the jenkinsci org found no usage of it.